src/js/editor/post.js

```diff
@@ -9,9 +11,24 @@ function isMarkupSection(section) {
   return section.type === MARKUP_SECTION_TYPE;
 }
 
+function isTextEditableSection(section) {
```

Comment:
Are the semantics here still meaningful, or perhaps this was intended to have other complexity?